src/components/carousel.js

```diff
@@ -0,0 +1,48 @@
+import { Swiper, SwiperSlide } from "swiper/react";
+import MovieCard from "./movie-card/movie-card";
+import SwiperCore, { Pagination, Navigation } from "swiper";
+SwiperCore.use([Pagination]);
+SwiperCore.use([Navigation]);
+
+export function Carousel(items) {
```

Comment:
The parameter passed isn't called `items`, it should be called `props`.

So instead of accessing `items.items` you should be calling `props.items`, and you can further reduce this nesting by destructuring like this:
```js
export function Carousel({ items }) {
  ...
}
```